catalyst/utils/config.py

```diff
@@ -50,8 +50,13 @@ def save_config(config, logdir: str) -> None:
 def parse_config_args(*, config, args, unknown_args):
```

Comment:
is it okay to finally write tests for this thing?
looks like it now DL stuff, so should be deterministic

Reply:
yeah sure